src/core.ts

```diff
@@ -0,0 +1,51 @@
+import * as dotenv from 'dotenv';
```

Comment:
It's better to name obvious, not like `core,share,lib` Here we can separate at least 3 components:
- config parsing
- config transformation
- config validation